### PR TITLE
Contact section

### PR DESCRIPTION
**Pull request by Leboroz**
### The section built in this milestone is:
The contact form in the footer:

Features added:
- Name field accepts up to 30 chars.
- Email field accepts only emails.
- Message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- A user cannot submit the form if one or more of the fields is empty.
- Collect data by using the Formspree service.

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -30,6 +26,19 @@ h1 {
   color: #fff;
 }
 
+.btn:hover {
```
Comment (carlos-ssh) [inline assets/css/styles.css:29]: - Provide the requested design. Add to your hover effect a color for your font inside your button.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @Leboroz , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/styles.css (hunk 1 above)
Comment (Leboroz) [reply assets/css/styles.css:29]: I did. there is a span inside the button.

Comment (carlos-ssh) [reply assets/css/styles.css:29]: Kindly add the 

required design to your button submit in your styleguide(Figma).

When you hover the button the design is missing the text.

Yours:

Screen Shot 2021-11-30 at 17 57 19

Styleguide:
Screen Shot 2021-11-30 at 18 02 45

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @Leboroz , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by lily-coder — APPROVED** [review]
Hi @Leboroz , 

Good job implementing the changes as requested by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**